packages/jaeger-ui/src/components/TracePage/SpanGraph/ViewingLayer.js

```diff
@@ -16,6 +16,7 @@
 
 import * as React from 'react';
 import cx from 'classnames';
+import { Button } from 'antd';
```

Comment:
Please sort the imports on the string portion.

Reply:
sorted. I also added a note for this for the future style guide. I'm guessing our sort is case-insensitive and `.` comes before `[a-zA-Z]`?

Reply:
More info about the sort of imports: https://github.com/jaegertracing/jaeger-ui/blob/style-guide-wip/CONTRIBUTING.md#imports

And, I forgot to mention (doh!) please have `import * as React from ...` as the first import.